winter/http/throttling.py

```diff
@@ -99,7 +96,8 @@ def create_throttle_classes(
 
```

Comment:
Remove `base_throttling_annotation = component.annotations.get_one_or_none(ThrottlingAnnotation)` line as it doesn't make sense any more.